save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -400,23 +435,14 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
 
         // fixme: table is rendered twice because of state change when `executionDto` is fetched
         testExecutionsTable {
-            status = state.status
-            testSuite = state.testSuite
+            status = state.filters.status
+            testName = state.filters.fileName
+            testSuite = state.filters.testSuite
+            tag = state.filters.tag
```

Comment:
Do you still want testExecutionsTable to have 4 different fields instead of one of type `Filters`?

Reply:
Done